cli/src/commands/operation/undo.rs

```diff
@@ -60,23 +62,31 @@ fn resets_view_of(op: &Operation, parent_op: &Operation) -> Result<bool, OpStore
     Ok(op.view_id() == grandparent_op?.view_id())
 }
 
+// Checks if `op` reverts the `maybe_bad_op` operation.
+fn is_undo_of(op: &Operation, maybe_bad_op: &Operation) -> bool {
+    op.metadata()
+        .description
+        .strip_prefix(UNDO_OPERATION_DESCRIPTION_PREFIX)
+        .is_some_and(|op_id| op_id == maybe_bad_op.id().hex())
+}
```

Comment:
nit: I think checking the description prefix is good enough, but if we require exact match, it's simpler to extract `tx_description(op) -> String` function? The extra formatting cost wouldn't matter.